draft-ietf-quic-multipath.md

```diff
@@ -1353,6 +1338,23 @@ Maximum Path Identifier:
   Receipt of a value that is higher than the local maximum value MUST
   be treated as a connection error of type PROTOCOL_VIOLATION.
 
+# Error Codes {#error-codes}
+
+Multipath QUIC transport error codes are 62-bit unsigned integers
+following {{QUIC-TRANSPORT}}.
+
+The following error codes are defined for use in the PATH_ABANDON frame.
```

Comment:
```suggestion
The following additional QUIC error codes are defined for use in the PATH_ABANDON frame.
```
I assume the intention is to be able also use e.g. NO_ERROR, right? Maybe it's good to say that explicitly?